FHIR/fhir-package-loader
# Tests for loader class and helper functions

**Description:**
This PR adds tests for the BasePackageLoader class. Tests are also added for the helper functions defined in `DefaultPackageLoader.ts`.

Additionally, due to my foolhardy clumsiness with git, I merged `main` into this branch, so this PR also includes the various documentation changes made during the transition to HL7.

**Testing Instructions:**
Run the tests locally and ensure they pass. Check the coverage report and verify that there is sufficient coverage in `BasePackageLoader.ts` and `DefaultPackageLoader.ts`. If there are test scenarios that should be added in order to cover more use cases, please let me know.

**Related Issue:**
N/A
